sql/hive/src/main/antlr3/org/apache/spark/sql/parser/SparkSqlParser.g

```diff
@@ -2386,8 +2386,8 @@ setColumnsClause
    KW_SET columnAssignmentClause (COMMA columnAssignmentClause)* -> ^(TOK_SET_COLUMNS_CLAUSE columnAssignmentClause* )
    ;
 
-/* 
-  UPDATE <table> 
+/*
+  UPDATE <table>
```

Comment:
For these white space things, maybe it is better to not change anything because it may be easy to cause conflict (if we are pretty active on updating the grammar).